### PR TITLE
Trial review Microverse PR

### PR DESCRIPTION
**Pull request by tarek102**
This PR is to review my trials project and to make any changes to it if required

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+body {
```
Comment (tarek102) [inline styles.css:1]: It would be better to fix linter errors like:
- Expected indentation of 2 spaces

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section id="main">
+        <h1 id="title">The Pyramids</h1>
+        <p class="cap">Impressive ancient structure</p>
+        <div id="img-div">
+            <img src="./images/pyramids-2371501-1920.jpg" id="image">
```
Comment (tarek102) [inline index.html:15]: It would be nice to add alt attribute to image tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            <caption id="img-caption">The pyramids are more than 7000 years old</caption>
```
Comment (tarek102) [inline index.html:16]: caption tag is Stray start tag “caption”

File: index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        </div>
+        <div id="tribute-info">
+            <h2 style="text-align: center;">Description</h2>
```
Comment (tarek102) [inline index.html:19]: Inline CSS should not be used in index.html
please remove it and add it to styles.css

**Review by tarek102 — COMMENTED** [review]
# Changes required ♻️ 

Please check my comments below and apply the changes.

**Review by KayLemba — APPROVED** [review]
Hi @tarek102 😃, 

You did a great job delivering such an amazing review 💯 Great summary 👍 Well done using the GitHub inline comments feature 👍 Polite and well-explained comments 👍

Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**